Sources/YapDB.swift

```diff
@@ -574,18 +580,18 @@ extension YapDB {
             query = q
         }
 
-        public convenience init(db: YapDatabase, view: YapDB.Fetch, query: Query) {
+        public convenience init(db: YapDatabase, view: YapDB.Fetch, query: @escaping Query) {
             self.init(db: db, views: [view], query: query)
         }
 
-        public func usingTerm(term: String) {
+        open func usingTerm(_ term: String) {
```

Comment:
I'm unsure why someone would want to override this, but I don't see the harm in it and I'd prefer to err on the side of extensibility when unsure.